src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,30 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+const NO_COMMENTS = 'No comments yet';
+
+export const PostInfo = ({ post }) => {
+  const { title, user, body, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <UserInfo user={user} />
```

Comment:
```suggestion
        {user && <UserInfo user={user} />}
```
user can be null, so better to add the check